docs/redirectless.md

```diff
@@ -0,0 +1,108 @@
+    var options = {
+      method: 'GET',
+      credentials: 'include'
+    }
+    return fetch(url, options);
+  }
+}
+```
+The `credentials: 'include'` Options attribute is required to make sure PingFederate's session cookie is handled correctly by the browser.
+
+### `onAuthorizationSuccess` function
+This callback function is designed to return the result of the transaction to the webpage containing the authentication API widget. The protocol response is passed to this function as the first argument when called by the authentication API widget.
+
+Here is an example: 
+```js
+var config = {
+  onAuthorizationSuccess: function (response) {
+    console.log(response.access_token);
+  }
+};
+```
+
+### Supported attributes
+The following attributes are supported by the internal onAuthorizationRequest function: `client_id`, `response_type`, `code_challenge`, `code_challenge_method`, `redirect_uri`, `scope`, `state`, `idp`, `pfidpadapterid`, `access_token_manager_id`, `aud`, `nonce`, `prompt`, `acr_values`, `max_age`, `login_hint`, `ui_locales`, `id_token_hint`, `claims_locales`. 
+
+The `client_id` and `response_type` attributes are required if `onAuthorizationRequest` is not present. 
+
+The present key-value pairs will be appended to the authorization request URL, the value will be URL encoded and arrays will be concatenated and URL encoded into one string.
+
+## Configuration object examples
+### OAuth 2.0 implicit
+This code snippet demonstrates a simple OAuth 2.0 configuration used by the Authentication API widget redirectless flow.
+```javascript
+var config = {
+  client_id: 'test',
+  response_type: 'token',
+  scopes: ['a', 'b', 'c'],
+  onAuthorizationSuccess: function (response) {
+    console.log(response.access_token);
+  }
+};
+```
+### OpenID Connect
+This code snippet demonstrates a simple OpenID Connect configuration used by the Authentication API widget redirectless flow.
+```javascript
+var config = {
+  scope: ['openid', 'profile', 'email', 'address', 'phone'],
+  state: '3f05dd88-3e97-496f-ba04-50e36e7ee1a5', // must be generated per each request.
+  client_id: 'test',
+  response_type: 'id_token',
+  onAuthorizationSuccess: function (response) {
+    console.log(response.id_token);
+  }
+};
+```
+## Starting webpack-dev-server with redirectless support
+Please follow these instructions to configure and start the webpack-dev-server with the redirectless support:
+
+1. Update the demo server's [index template](../demo-server/templates/index-template.handlebars) file with a configuration object. Configuration object examples are available at the [Configuration object examples](#configuration-object-examples) section of this document.
+1. The following command should be executed to start the webpack-dev-server and initialize the widget in redirectless mode:
+```bash
+OPERATIONMODE=redirectless npm run start
+```
+1. Use your browser to go to https://localhost:8080 to see the widget in action.
```

Comment:
also the count here is started at 1. again, I am assuming you intended for it to be 3.?